calculator.go

```diff
@@ -3,25 +3,75 @@ package calculator
 
 import (
 	"errors"
+	"math"
+	"strconv"
+	"strings"
 )
 
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
 	return a + b
 }
 
+// AddN takes a slice of numbers and returns the addition of the given numbers
+func AddN(numbers []float64) float64 {
```

Comment:
This is good, but it's not quite what the exercise is asking for, which is a function that accepts _two or more_ inputs. For example, `Divide(12, 4, 3)`. That wouldn't work here, would it?

Reply:
Yes, somehow in my head I made it bigger problem for n numbers.